src/context/auth.jsx

```diff
@@ -1,156 +1,167 @@
-    try {
-      const res = await login(email, password);
-
-      if (res.data.error) {
-        setError(res.data.error);
-        return;
-      }
-
-      if (res.data.token) {
-        localStorage.setItem("token", res.data.token);
-        setToken(res.data.token);
-        navigate(location.state?.from?.pathname || "/");
-        setError("");
-        return;
-      }
-      setUser({...res.data.user})
-      setError("Login failed");
-      navigate("/login");
-    } catch (error) {
-      setError("An error occurred during login");
+
+    const handleLogout = () => {
+        localStorage.removeItem('token')
+        setToken(null)
+        setUser(null)
+        setError('')
     }
-  };
-
-  const handleLogout = () => {
-    localStorage.removeItem("token");
-    setToken(null);
-    setUser(null)
-    setError("");
-  };
-
-  const handleRegister = async (email, password) => {
-    try {
-      const res = await register(email, password);
-      if (res.data.error) {
-        setError(res.data.error);
-        return;
-      }
-      localStorage.setItem("token", res.data.token);
-      setToken(res.data.token);
-      navigate("/verification");
-      setError("");
-    } catch (error) {
-      setError("An error occurred during registration");
+
+    const handleRegister = async (email, password) => {
+        try {
+            const res = await register(email, password)
+            if (res.data.error) {
+                setError(res.data.error)
+                return
+            }
+            localStorage.setItem('token', res.data.token)
+            setUser({ ...res.data.user })
+            setToken(res.data.token)
+            navigate('/verification')
+            setError('')
+        } catch (error) {
+            setError('An error occurred during registration')
+        }
     }
-  };
-
-  const handleCreateProfile = async (
-    firstName,
-    lastName,
-    username,
-    githubUsername,
-    mobile,
-    bio
-  ) => {
-    try {
-      const { userId } = jwt_decode(token);
-
-      const res = await createProfile(
-        userId,
+
+    const handleCreateProfile = async (
         firstName,
         lastName,
         username,
         githubUsername,
         mobile,
         bio
-      );
-
-      if (res.status === "success") {
-        navigate("/");
-        return;
-      }
-      setError("Failed to create profile");
-    } catch (error) {
-      setError("An error occurred during profile creation");
+    ) => {
+        try {
+            const { userId } = jwt_decode(token)
+
+            const res = await createProfile(
+                userId,
+                firstName,
+                lastName,
+                username,
+                githubUsername,
+                mobile,
+                bio
+            )
+
+            if (res.status === 'success') {
+                navigate('/')
+                return
+            }
+            setError('Failed to create profile')
```

Comment:
I think we should move these strings to a file like you did with the backend, it will be useful when internationalization comes into play

Reply:
Sure! These aren't new with this commit, should we make a new ticket and address this there then? Or should I just tack it on here?

Reply:
Good point, let's start a new piece of work for them